swap/cashout_test.go

```diff
@@ -132,12 +173,14 @@ func TestCashCheque(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = cashoutProcessor.cashCheque(context.Background(), &CashoutRequest{
+	cashoutProcessor.submitCheque(context.Background(), &CashoutRequest{
```

Comment:
At some point there is nothing else we can do other than log the error. This way we have all the cashing-related error handling in one place. otherwise we end up once again handling some cashing errors in swap and others here.

Reply:
can't argue with that.